File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 4/3/2026, 2:00:51 PM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        actualSum = sum(nums)
        n = len(nums)
        totalSum = (n*(n+1)) // 2
        return totalSum - actualSum

File: 268-MissingNumber/268-MissingNumber.cpp
// Last updated: 4/3/2026, 2:00:47 PM
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int actualSum = 0, expectSum = 0;
        for (int num : nums)
            actualSum += num;
        expectSum = (n * (n + 1)) / 2;
        return expectSum - actualSum;
    }
};